src/common/options/rgw.yaml.in

```diff
@@ -3556,3 +3556,10 @@ options:
   - rgw
   - osd
   with_legacy: true
+- name: rgw_allow_secrets_in_cleartext
+  type: bool
+  level: advanced
+  desc: Allows sending secrets (e.g. passwords) over non encrypted HTTP messages
```

Comment:
please use an option name that's specific to pubsub or notifications

in the description, please add some strong wording that this is a security vulnerability and should never be used in production

this is related to `rgw_trust_forwarded_https`, so a `see_also:` would be helpful

Reply:
@cbodley please see: https://github.com/ceph/ceph/pull/43436/files#diff-32e4985985c3a5adaddfe7a70c31a1112eb5887cd1149a967dac5bc303487903R3566-R3569

let me know if this is enough.